controllers/fenceagentsremediation_controller.go

```diff
@@ -173,6 +174,17 @@ func (r *FenceAgentsRemediationReconciler) Reconcile(ctx context.Context, req ct
 		r.Log.Info("Finalizer was removed", "CR Name", req.Name)
 		return emptyResult, nil
 	}
+
+	// Validate whetehr the fence agent is supported
+	r.Log.Info("Check if the agent is supported")
+	if err := r.checkAgentSupport(far.Spec.Agent); err != nil {
+		updateErr := utils.UpdateConditions(utils.FenceAgentNotSupported, far, r.Log)
+		if updateErr != nil {
+			err = fmt.Errorf("validion error for fence agent: %w, and status error: %w", updateErr, err)
+		}
+		return emptyResult, err
```

Comment:
IIUC no value in returning an error here since there isn't a chance that this will be resolved in another reconcile

Reply:
> since there isn't a chance that this will be resolved in another reconcile

The only chance is by updating the fence agent's name, but still, we can potentially print once and not multiple times.

Reply:
Good point,
However IIUC it's pretty unlikely, since this name is retrivied from the CR which is created automatically by NHC and not user created.
it is much more likely that the fence agent name will be changed in the Template or Configuration (I can't remember which one is the origin :sweat_smile: )

Reply:
>created automatically by NHC and not user created.

fine by me.

>Template or Configuration (I can't remember which one is the origin 😅 )

Just template.